docker/test

```diff
@@ -9,19 +9,23 @@ HERE=$(dirname $0)
 NAME_SERVER=mintr_server
 
 function cleanup {
+    set +e
+    echo "Logs"
+    docker logs $NAME_SERVER
     echo "Cleaning up"
-    docker kill $NAME_SERVER > /dev/null || true
+    docker kill $NAME_SERVER > /dev/null
+    docker rm -f $NAME_SERVER > /dev/null
 }
 
 trap cleanup EXIT
 
-docker run --rm -d \
+docker run -d \
```

Comment:
What was the need to take out `--rm` and does that mean the container should be taken down later?

Reply:
The --rm was so that I could get logs out of the container before removing it (even on failure). It is removed in the catch